src/pages/Settings/UICard.tsx

```diff
@@ -174,7 +174,7 @@ const UICard = () => {
           />
         </div>
       )}
-      {showFeatures.integrations && (
```

Comment:
@Mattallmighty not yet bro. I need to fix the "progressing seconds handling" first